packages/react-components/src/trace-explorer/trace-explorer-placeholder-widget.tsx

```diff
@@ -16,19 +22,26 @@ export class ReactExplorerPlaceholderWidget extends React.Component<ReactPlaceho
         return (
             <div className="placeholder-container" tabIndex={0}>
                 <div className="center">{'You have not yet opened a trace.'}</div>
-                <div className="placeholder-open-workspace-button-container">
-                    <button
-                        className="plcaeholder-open-workspace-button"
-                        title="Select a trace to open"
-                        onClick={this.props.handleOpenTrace}
-                        disabled={this.props.loading}
-                    >
-                        {this.props.loading && <FontAwesomeIcon icon={faSpinner} spin style={{ marginRight: '5px' }} />}
-                        {this.props.loading && <span>Connecting to trace server</span>}
-                        {!this.props.loading && <span>Open Trace</span>}
-                    </button>
-                </div>
+                {this.renderButton(TraceResourceType.FOLDER, 'Open Trace Folder')}
+                {this.renderButton(TraceResourceType.FILE, 'Open Trace File')}
             </div>
         );
     }
+
+    renderButton(type: TraceResourceType, title: string): React.ReactNode {
+        return this.props.renderOnly === undefined || this.props.renderOnly === type ? (
+            <div className="placeholder-open-workspace-button-container">
+                <button
+                    className="plcaeholder-open-workspace-button"
+                    title="Select a trace to open"
```

Comment:
This should be different for each case to be clear to the user, maybe 'Select a single trace file to open' for file traces and 'Select a trace folder to open`

Reply:
Ok - I will update it in the next patch!